### ISSUE TITLE
Implement React Dom Server into the types library

### ISSUE THREAD

**Issue by Soremwar** [denoland/deno#4722]
The following issue tracks the progress fo adding a definition file for React Dom Server library into the std/types library,

- [ ] Implement compatible functions with Deno ecosystem (renderToString, renderToStaticMarkup). ~(Waiting on #4713)~ #4768
- [ ] Implement Readable Streams in std/node library.
- [ ] Implement not compatible functions with native Deno ecosystem (renderToNodeStream, renderToStaticNodeStream).

**Comment by Soremwar** [comment]
@kitsonk Can you assign me this issue please? :sweat_smile:

**Comment by Soremwar** [comment]
Closing in favor of https://github.com/Soremwar/deno_types/issues/1
